server.js

```diff
@@ -1 +1,9 @@
-module.exports = function () {}
+if (isElectron()) {
+  module.exports = require('.')
+} else {
+  module.exports = function () {}
+}
+
+function isElectron () {
+  return process && process.versions && (process.versions.electron !== undefined)
```

Comment:
We should also detect if its running in the main process or a browser window.

Reply:
```js
function isElectronRenderer () { 
  return process && process.type && process.type === 'renderer'
}
```

Reply:
Ooh good point. I'll fix this during lunch. 😊